internal/runners/projects/edit.go

```diff
@@ -0,0 +1,179 @@
+		if strings.EqualFold(params.Visibility, visibilityPublic) {
+			editable.Private = p.BoolP(false)
+		} else {
+			editable.Private = p.BoolP(true)
+		}
+	}
+
+	if params.Repository != "" {
+		editMsg += locale.Tl("edit_prompt_repo", "  - Repository: {{.V0}}\n", params.Repository)
+		editable.RepoURL = &params.Repository
+	}
+	editMsg += locale.Tl("edit_prompt_confirm", "Continue?")
+
+	edit, err := e.prompt.Confirm("", editMsg, p.BoolP(false))
+	if err != nil {
+		return locale.WrapError(err, "err_edit_prompt", "Could not prompt for edit confirmation")
+	}
+
+	if !edit {
+		e.out.Print(locale.Tl("edit_cancelled", "Project edit cancelled"))
+		return nil
+	}
+
+	if err = model.EditProject(params.Namespace.Owner, params.Namespace.Project, editable); err != nil {
+		return locale.WrapError(err, "err_edit_project", "Could not edit project")
+	}
+
+	if err = e.editLocalCheckouts(params); err != nil {
+		return locale.WrapError(err, "err_edit_local_checkouts", "Could not edit local checkouts")
+	}
+
+	if err = e.updateProjectMapping(params); err != nil {
+		return locale.WrapError(err, "err_edit_project_mapping", "Could not update project mapping")
+	}
+
+	e.out.Print(locale.Tl("edit_success", "Project edited successfully"))
+
+	return nil
+}
+
+func (e *Edit) editLocalCheckouts(params EditParams) error {
+	localProjects := projectfile.GetProjectMapping(e.config)
+
+	var localCheckouts []string
+	for namespace, checkouts := range localProjects {
+		if namespace == params.Namespace.String() {
+			localCheckouts = append(localCheckouts, checkouts...)
+		}
+	}
+
+	for _, checkout := range localCheckouts {
+		err := e.editLocalCheckout(params.Namespace.Owner, checkout, params)
+		if err != nil {
+			return errs.Wrap(err, "Could not edit local checkout at %s", checkout)
+		}
+	}
+
+	return nil
+}
+
+func (e *Edit) editLocalCheckout(owner, checkout string, params EditParams) error {
+	if params.ProjectName == "" {
+		return nil
+	}
+
+	pjFile, err := projectfile.FromPath(checkout)
+	if err != nil {
+		return errs.Wrap(err, "Could not get projectfile at %s", checkout)
+	}
+
+	err = pjFile.SetNamespace(owner, params.ProjectName)
+	if err != nil {
+		return errs.Wrap(err, "Could not set project namespace at %s", checkout)
+	}
+
+	return nil
+}
+
+func (e *Edit) updateProjectMapping(params EditParams) error {
+	if params.ProjectName == "" {
+		return nil
+	}
+
+	localProjects := projectfile.GetStaleProjectMapping(e.config)
+
+	var localCheckouts []string
+	for namespace, checkouts := range localProjects {
+		if namespace == params.Namespace.String() {
+			localCheckouts = append(localCheckouts, checkouts...)
+		}
+	}
+
+	ns, err := project.ParseNamespace(fmt.Sprintf("%s/%s", params.Namespace.Owner, params.ProjectName))
+	if err != nil {
+		return errs.Wrap(err, "Could not parse namespace")
+	}
+
+	for _, checkout := range localCheckouts {
+		projectfile.StoreProjectMapping(e.config, ns.String(), checkout)
+	}
```

Comment:
Again, can these two loops be combined? It just feels like extra code for no reason. I could be wrong though.